functions.lua

```diff
@@ -259,6 +259,7 @@ function telemosaic.rightclick(pos, node, player, itemstack, pointed_thing)
 	if player.is_fake_player or not minetest.is_player(player) then
 		return itemstack  -- No fake players!
 	end
+	if not itemstack then return end
```

Comment:
I think you could condense this into
```suggestion
	if not itemstack or player.is_fake_player or not minetest.is_player(player) then
		return itemstack  -- No fake players!
	end
```